discovery/gossiper_test.go

```diff
@@ -837,3 +837,154 @@ func TestOrphanSignatureAnnouncement(t *testing.T) {
 		t.Fatal("wrong number of objects in storage")
 	}
 }
+
+// TestDeDuplicatedAnnouncements ensures that the deDupedAnnouncements
+// struct properly stores and delivers the set of de-duplicated
+// announcements.
+func TestDeDuplicatedAnnouncements(t *testing.T) {
```

Comment:
As this test is completely independant of the other tests, and doesn't share any global state it should be run in parallel. We can achieve this by adding `t.Parallel()` at the very top of this test. Thanks!